src/transformState.js

```diff
@@ -4,8 +4,32 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
You can make destructuring to get all needed properties of action and make your code cleaner

Reply:
Pushed. Believ inderstood you well